frontend/src/_styles/theme.scss

```diff
@@ -1877,6 +1877,18 @@ button {
   border-color: none;
 }
 
+.form-check-input {
+  cursor: pointer;
+}
+
+.show-password-label {
+  width: fit-content;
+ }
+
+ .form-checks {
```

Comment:
please use appropriate names for classes
like `.show-password-field` for` .show-password-label`
and `.show-password-label` for ` form-checks` 

And can we put the .show-password-label class inside .show-password-field ?
like 
```
.show-password-field {
  .show-password-label {
  }
 }
```

Reply:
Hey @shah21 I have created another PR #3428  please review If you satisfy with that I will close this PR.